assets/src/application/ui/display/EntityListLegend/index.tsx

```diff
@@ -9,10 +10,15 @@ const EntityListLegend: React.FC<EntityListLegendProps> = ({ legendConfig }) =>
 	const { icons, swatches } = legendConfig;
 
 	const iconsSource = icons.map(({ icon, description }) => {
+		// @ts-ignore
+		const term = (!is(String)(icon) && React.createElement(icon)) || (
```

Comment:
not the cleanest code I've written, but will have to keep this until will transition to SVGs